### PR TITLE
Ibm20180703 12042

### PR DESCRIPTION
**Pull request by ScottMooreIBM**

### CODE REVIEW

**Comment by ScottMooreIBM** [comment]
Oversight due to such a large group of similar vulnerabilities.  We will fix.

**Comment by cve-team** [comment]
All done now?

**Comment by ScottMooreIBM** [comment]
We think so.

File: 2017/1xxx/CVE-2017-1293.json [hunk 1]
```diff
@@ -1,17 +1,147 @@
...
+            "S" : "C",
+            "UI" : "R",
+            "A" : "N"
+         }
+      }
+   },
+   "description" : {
+      "description_data" : [
+         {
+            "value" : "IBM Rational Quality Manager and IBM Rational Collaborative Lifecycle Management 5.0 through 5.0.2 and 6.0 through 6.0.5 are vulnerable to cross-site scripting. This vulnerability allows users to embed arbitrary JavaScript code in the Web UI thus altering the intended functionality potentially leading to credentials disclosure within a trusted session.  IBM X-Force ID:  124759",
```
Comment (cve-team) [inline 2017/1xxx/CVE-2017-1293.json]: The X-Force id in the description should be 125154, no?

**Comment by cve-team** [comment]
NB: I noticed the same sort of problem with CVE-2017-1280 and have fixed that on our end.

**Pull request merged**